tests/kerastuner/engine/tuner_workflows_test.py

```diff
@@ -178,9 +180,9 @@ def test_override_compile(tmp_dir):
         directory=tmp_dir)
 
     assert tuner.oracle.objective.name == 'val_mse'
-    assert tuner.optimizer == 'rmsprop'
-    assert tuner.loss == 'sparse_categorical_crossentropy'
-    assert tuner.metrics == ['mse', 'accuracy']
+    assert tuner.hypermodel.optimizer == 'rmsprop'
```

Comment:
You'd expect the class that was passed `optimizer` etc to hold these attributes, e.g. `x = X(foo=bar); x.foo == bar`

Reply:
In general agreed but we're already breaking that assumption for our convenience `Tuner`s which accept args that are passed to the `Oracle`. 

We could optionally mirror these here, but if so we should make them properties that refer to the `KerasModelWrapper` properties so that changing them changes the corresponding values in the `KerasModelWrapper`. WDYT?